doc/sphinx/source/install.rst

```diff
@@ -87,7 +88,7 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
 
    .. code-block:: console
 
-      $ cmake ../ [-DNGRAPH_USE_PREBUILT_LLVM=TRUE]
+      $ cmake ../ [-DNGRAPH_USE_PREBUILT_LLVM=TRUE] [-DNGRAPH_TARGET_ARCH=skylake-avx512]
```

Comment:
I _think_ this `-DNGRAPH_TARGET_ARCH` flag will only work on Skylake machines. Do we specify that? (If it's not specified, you might check with @jbobba to make sure I'm not wrong)